rest/route/github.go

```diff
@@ -266,7 +278,11 @@ func (gh *githubHookApi) Run(ctx context.Context) gimlet.Responder {
 		}
 
 		baseBranch := *pr.Base.Ref
-		err = gh.sc.EnqueueItem(userRepo.Owner, userRepo.Repo, baseBranch, strconv.Itoa(PRNum))
+		item := commitqueue.CommitQueueItem{
+			Issue:   strconv.Itoa(PRNum),
+			Modules: commentData.Modules,
+		}
+		err = gh.sc.EnqueueItem(userRepo.Owner, userRepo.Repo, baseBranch, item)
```

Comment:
seems weird that we'd need to depend on the commitqueue package to make calls to the data connector, is there a way to remove that dependency?

Reply:
removed the dependency.
Now depends on the rest model.